src/ActionList/List.tsx

```diff
@@ -82,6 +87,12 @@ export type ListProps = ListPropsBase | GroupedListProps
 
 const StyledList = styled.div`
   font-size: ${get('fontSizes.1')};
+  /* 14px font-size * 1.428571429 = 20px line height
+   *
+   * TODO: When rem-based spacing on a 4px scale lands, replace
+   * hardcoded '1.428571429'
+   */
+  line-height: 1.428571429;
```

Comment:
Could we just hard code 20px here instead?
```suggestion
  line-height: 20px;
```

Reply:
@smockle thoughts on this?

Reply:
I’m fine either way: The hardcoded value is going to be replaced with a variable after the [“new scaling and typography internal variables”](https://github.com/github/primer/issues/41#issuecomment-839264461) land. cc @vdepizzol for visibility

Reply:
Thanks @smockle.  I made the switch to `20px` since it's a little easier to read for now.